project_claims/i18n/en_US.po

```diff
@@ -1,19 +1,21 @@
 # Translation of OpenERP Server.
```

Comment:
I didn't put new translations in, just synced the existing ones with the build... anyway with the new commit the translations (en, en_US) you mentioned were removed and templates put at their place.